frontend/src/routes/(authenticated)/project/create/+page.svelte

```diff
@@ -153,6 +164,20 @@
 
     <ProjectTypeSelect bind:value={$form.type} error={$errors.type} />
 
+    <DevContent>
+      <Select
+        id="org"
+        label={$t('project.create.org')}
+        bind:value={$form.orgId}
+        error={$errors.orgId}
+        on:change
+      >
+        {#each myOrgs as org}
+          <option value={org.id}>{org.name}</option>
```

Comment:
In this [commit](https://github.com/sillsdev/languageforge-lexbox/pull/909/commits/f0dfe69ad057d27518ebbce0c6deb82dafe4cff9), I created a null option named 'Select an organization' that is selected by default for admin users. However, non-admin users who have their first organization selected will have to choose 'Select an organization' as a null option if they want to deselect an organization, which might be confusing. We could either:

1. Keep 'Select an organization'.
2. Leave it blank, which may better communicate the idea of a null selection.

Reply:
I would prefer to change it to be 'No organization', otherwise when the first org is selected by default, if the user wants it to not be owned by an org it's kinda confusing to select the 'Select an org' option.